two-step-view/README.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: pattern
+title: Two-step view 
+folder: two-step-view
+permalink: /patterns/two-step-view/
+categories: Structural
+language: en
+tags:
+- Creational
+---
```

Comment:
Please implement the frontmatter according to the instructions at https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute